content/docs/instrumenting/exporters.md

```diff
@@ -88,6 +88,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [MQTT blackbox exporter](https://github.com/inovex/mqtt_blackbox_exporter)
    * [RabbitMQ exporter](https://github.com/kbudde/rabbitmq_exporter)
    * [RabbitMQ Management Plugin exporter](https://github.com/deadtrickster/prometheus_rabbitmq_exporter)
+   * [IBM MQ metrics sample](https://github.com/ibm-messaging/mq-metric-samples)
```

Comment:
Can you link to the actual Prometheus-relevant bit?

Reply:
thanks for your reply! i have made some changes according to your guide, this repo is supplied by the ibm-messaging team, maybe i could pass on your coding suggestion to them!